pet_rock.py

```diff
@@ -0,0 +1,66 @@
+def naming():
+    name = input("What would you like to name your pet rock?")
+    return name
+def show_stats(name,rock):
+    print(f"great choice your rocks name is {name}")
+    print(f"Hunger:{rock['Hunger']}/10")
+    print(f"Happiness:{rock['Happiness']}/10")
+
+def main():
+
+    rock = { 
+        "Hunger" : 5,
+        "Happiness" : 5,
+        }
+    
+    print("Welcome to the Pet Rock Simulator!")
+
+    name = naming()
+
+    while True:
+        show_stats(name, rock)
+        if rock["Hunger"] >= 10:
+            print(f"{name} got too hungry and rolled away to find food")
+            print("Game Over")
+            break 
+        if rock["Happiness"] <= 0:
+            print(f"{name} got to sad and left you for a owner who is more fun.")
+            print("Game Over")
+            break 
+        print("What do you want to do with your Rock?")
+        print(" 1. Feed the rock")
```

Comment:
```suggestion
        print(" 1. Feed the rock\n"
        "2.\n"
        "3.\n")
```
this can be used to simplify print command